### PR TITLE
Add home work 1

### PR DESCRIPTION
**Pull request by DimaKobets**

### CODE REVIEW

File: internship2018-1/src/assets/bundle.js [hunk 1]
```diff
@@ -0,0 +1,112 @@
+/******/ (function(modules) { // webpackBootstrap
```
Comment (vkhmelko-softheme) [inline internship2018-1/src/assets/bundle.js]: please remove this file from repo. Because it's autogenerated

**Review by vkhmelko-softheme — CHANGES_REQUESTED** [review]

**Comment by DimaKobets** [comment]
bundle is removed

**Comment by vkhmelko-softheme** [comment]
320 files changed! Not bad, not bad :)

Any chance to use npm and webpack to launch the project?

**Comment by vkhmelko-softheme** [comment]
xina-resolutions

Your home task looks pretty good. Seems you know some html and css. Do you have any experience with bootstrap? Because what I need is only one item on the line for mobile devices and only 3 items for tablets and lager screens. 

Something line this:
fireshot capture 3 - hello world - http___localhost_8081_products html
fireshot capture 4 - hello world - http___localhost_8081_products html

Try to use bootstrap. I'm sure it will be easy for you

**Review by vkhmelko-softheme — APPROVED** [review]

File: Homeworks/HW 2/filterArray.js [hunk 2]
```diff
@@ -0,0 +1,81 @@
...
+        }            
+    }
+    return true;
+}
+
+function showMessage (expression){
+    if (expression){
+        console.log("object is empty")
+    }
+    else{
```
Comment (dchecherynda-softheme) [inline Homeworks/HW 2/filterArray.js:44]: The else statement should be placed on the same line as the closing curly brace:

if (expression) {
    ...
} else {
    ...
}

File: Homeworks/HW 2/filterArray.js [hunk 3]
```diff
@@ -0,0 +1,81 @@
...
+// Task 3 ____________________
+console.log("Task - 3");
+
+function keysOfObject (obj){
+
+    var arrayOfKeys = [];
+    var i = 0;
+
+    for (var key in obj){
+        arrayOfKeys[i] = obj[key];
```
Comment (dchecherynda-softheme) [inline Homeworks/HW 2/filterArray.js:66]: The **arrayofKeys** should contain keys of an object, not values - arrayOfKeys[i] = key;

**Pull request merged**